code/index.html

```diff
@@ -2,21 +2,85 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Quarantine Times</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
-  </head>
-  <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
-
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
-
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
+      </head>
+      <body>
+
+<header class="header1">
+  <div class="hamburger">
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
+  </div>
+
+    <img class="logo-pic" src="qllogo.png" alt="logo">
+    <h1 class="qtimes">Quarantine Times</h1>
```

Comment:
Since these two are "siblings" to thi div and the nav, they should not be indented